fe/fe-core/src/main/java/org/apache/doris/analysis/ShowUserStmt.java

```diff
@@ -28,9 +28,8 @@ public class ShowUserStmt extends ShowStmt {
 
     static {
         ShowResultSetMetaData.Builder builder = ShowResultSetMetaData.builder();
-        for (String title : AuthProcDir.TITLE_NAMES) {
-            builder.addColumn(new Column(title, ScalarType.createVarchar(30)));
-        }
+        builder.addColumn(new Column("User", ScalarType.createVarchar(30)));
```

Comment:
We should show UserIdentity to user, just like Mysql.
For example, `'root'@'localhost'.`

Reply:
OK, it will may be the same as `select user()` stmt.